app/views/properties/_show.html.haml

```diff
@@ -1,18 +1,17 @@
-= render TurboFrameComponent.new(id: 'property_show', data: {"turbo-frame-target": "frame"}) do
+= render TurboFrameComponent.new(id: 'property_show') do
   - if @property
     - if @property.errors
       = render Display::AlertComponent.new(type:'info', message: @property.errors.join)
     - else
       - properties = LinkedData::Client::Models::Property.properties_to_hash(@property).first
-      = render ConceptDetailsComponent.new(id:'property-details', acronym: @acronym, concept_id: @property.id,
-                      properties: @property.properties,
-                      top_keys: [],
-                      bottom_keys: properties.keys.map(&:to_s),
-                      exclude_keys: []) do |c|
-        - c.header(stripped: true) do |t|
-          - t.add_row({th: t('properties.id')}, {td: link_to_with_actions(@property.id, acronym: @acronym)})
-          - t.add_row({th: t('properties.type')}, {td: @property.type })
-          - t.add_row({th: t('properties.preferred_name')}, {td: display_in_multiple_languages(@property.label)}) unless  @property.label.blank?
-          - t.add_row({th: t('properties.definitions')}, {td: display_in_multiple_languages(@property.definition)}) unless @property.definition.blank?
-          - t.add_row({th: t('properties.domain')}, {td:  get_link_for_cls_ajax(@property.domain, @acronym, '_top')}) unless @property.domain.blank?
-          - t.add_row({th: t('properties.range')}, {td: get_link_for_cls_ajax(@property.range, @acronym, '_top')}) unless @property.range.blank?
+      = render TabsContainerComponent.new(type:'outline') do |c|
+        - apikey = "#{get_apikey}"
```

Comment:
this code is repeated at least two times, think of extracting it as a view component

Reply:
same remark